samples/teacherNotebookA.dib

```diff
@@ -1,22 +1,43 @@
 #!markdown
 
-# Stage 1 (students wont see this line)
+# Stage 1
 
 #!markdown
 
-## [Set up] (students wont see this line)
+## [SetUp]
 
 #!csharp
 
 double GetConstant() => 1.0;
 
+#!csharp
+
+Lesson.NextEvaluation.AddRule(
```

Comment:
Would `Lesson.Current.AddRule` be clearer?

Reply:
`Current` seems to refer to the current lesson. Perhaps `CurrentEvaluation` or `CurrentStage`?

Reply:
We have `Kernel.Current` because `Kernel` is both a class that can be instantiated and one that has a current instance in the context of a code submission. I'm guessing `Lesson` could be similar.

"Evaluation" is what we've been calling the result of the rules and is currently just a value object with no behavior.

Reply:
My understanding is that lesson refers to the documents (teacher and student notebooks). So how does `Stage.Current` sound?